apps/bfd-server/bfd-server-war/src/test/resources/openapi/posts.yaml

```diff
@@ -354,3 +354,241 @@ paths:
               schema: { $ref: '#/components/schemas/FHIR-JSON-RESOURCE' }
             application/fhir+xml:
               schema: { $ref: '#/components/schemas/FHIR-XML-RESOURCE' }
+  /v2/fhir/Claim/_search:
+      post:
+        tags: [ v2,Claim ]
+        summary: 'search-type: Search for Claim instances'
+        description: This is a search type
+        requestBody:
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  mbi:
+                    type: string
+                    description: |-
+                      **NOTE: THIS IS A REQUIRED FIELD**
+
+                      Fetch _Beneficiary_ data using a FHIR _MBI_ identifier; an MBI
+                      represents the medicare benficiary ID, or as much of that
+                      defined by the [FHIR specification](https://terminology.hl7.org/NamingSystem-cmsMBI.html) as:
+
+                      ```
+                      A combination of upper case ASCII letters ('A'..'Z except for S, L, O, I, B, and Z.', numerals ('0'..'9'),
+                      '-', with a length limit of 11 characters.
+                      ```
+
+                      The following are all valid values for MBI, and all might
+                      represent the same resource:
+                        - `mbi=9AB2WW3GR44` (unhashed MBI)
+                        - `mbi=82273caf4d2c3b5a8340190ae3575950957ce469e593efd7736d60c3b39d253c` (hashed)
```

Comment:
I haven't looked _too_ closely at the implementation, but the spirit of enabling (and documenting) the POST endpoints is, in part, to discourage the use of this hashed identifier under RFC20.

* Are hashed MBIs honored here?
* If they are accepted, would it be better to omit them from this documentation _or_ somehow label them as deprecated?

Reply:
removed hashed mbis

Reply:
Hashed mbis still work on POST with the `isHashed` flag `true`